### PR TITLE
[zh-cn]: sync translation for `BatteryManager`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the https://github.com/mdn/translated-content/issues/18191

depends on https://github.com/mdn/content/pull/32218

see also https://developer.mozilla.org/en-US/docs/Web/API/BatteryManager

### CODE REVIEW

File: files/zh-cn/web/api/batterymanager/index.md [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
 slug: Web/API/BatteryManager
+l10n:
+  sourceCommit: 8d54a21ae2677dba11569e7b7d918eac828af0b3
 ---
 
-{{APIRef}}
+{{ApiRef("Battery API")}}{{securecontext_header}}
 
-[电池状态 API](/zh-CN/docs/Web/API/Battery_Status_API) 中的 `BatteryManager` 接口提供了系统电池电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，该 promise 在完成状态时将返回一个 `BatteryManager` 接口。
+{{domxref("Battery Status API", "", "", "nocode")}} 的 **`BatteryManager`** 接口提供了系统电源电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，其兑现一个 `BatteryManager` 对象。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:10]:
```suggestion
{{domxref("Battery Status API", "", "", "nocode")}} 的 **`BatteryManager`** 接口提供了系统电池电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，其兑现一个 `BatteryManager` 对象。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 2]
```diff
@@ -1,39 +1,47 @@
...
 {{InheritanceDiagram}}
 
-## 属性
+## 实例属性
+
+_也从其父接口 {{domxref("EventTarget")}} 继承属性。_
 
 - {{domxref("BatteryManager.charging")}} {{ReadOnlyInline}}
-  - : 一个布尔值，说明当前电池是否正在充电。
+  - : 一个指示当前电源是否正在充电的布尔值。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:21]:
```suggestion
  - : 一个指示当前电池是否正在充电的布尔值。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 3]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.chargingTime")}} {{ReadOnlyInline}}
-  - : 一个数字，代表距离充电完毕还需多少秒，如果为 0 则充电完毕。
+  - : 一个表示距离充电完毕还需多少秒的数字，或为 0 若电源已充电完毕。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:23]:
```suggestion
  - : 一个表示电池距离充电完毕还需多少秒的数字，或为 0 则表示电池已充电完毕。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 4]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.dischargingTime")}} {{ReadOnlyInline}}
-  - : 一个数字，代表距离电池耗电至空且系统挂起需要多少秒。
+  - : 一个表示距离电源耗电至空且系统挂起需要多少秒的数字。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:25]:
```suggestion
  - : 一个表示距离电池耗电至空且系统挂起需要多少秒的数字。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 5]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.level")}} {{ReadOnlyInline}}
-  - : 一个数字，代表系统电量的水平，这个值放缩在 0.0 至 1.0 之间。
+  - : 一个表示系统的电源电量水平的数字，其值介于 0.0 和 1.0 之间。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:27]:
```suggestion
  - : 一个表示系统的电池电量水平的数字，其值介于 0.0 和 1.0 之间。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 6]
```diff
@@ -1,39 +1,47 @@
...
 
-_方法继承自其父接口_：{{domxref("EventTarget")}}。
+## 事件
 
-### 事件
+_也从其父接口 {{domxref("EventTarget")}} 继承事件。_
 
 - {{domxref("BatteryManager.chargingchange_event", "chargingchange")}}
-  - : 当电池充电状态（{{domxref("BatteryManager.charging", "charging")}} 属性）更新时被触发。
+  - : 当电源充电状态（{{domxref("BatteryManager.charging", "charging")}} 属性）更新时触发。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:38]:
```suggestion
  - : 当电池充电状态（{{domxref("BatteryManager.charging", "charging")}} 属性）更新时触发。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 7]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.chargingtimechange_event", "chargingtimechange")}}
-  - : 当电池充电时间（{{domxref("BatteryManager.chargingTime", "chargingTime")}} 属性）更新时被触发。
+  - : 当电源充电时间（{{domxref("BatteryManager.chargingTime", "chargingTime")}} 属性）更新时触发。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:40]:
```suggestion
  - : 当电池充电时间（{{domxref("BatteryManager.chargingTime", "chargingTime")}} 属性）更新时触发。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 8]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.dischargingtimechange_event", "dischargingtimechange")}}
-  - : 当电池断开充电的时间（{{domxref("BatteryManager.dischargingTime", "dischargingTime")}} 属性）更新时被触发。
+  - : 当电源续航时间（{{domxref("BatteryManager.dischargingTime", "dischargingTime")}} 属性）更新时触发。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:42]:
```suggestion
  - : 当电池续航时间（{{domxref("BatteryManager.dischargingTime", "dischargingTime")}} 属性）更新时触发。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 9]
```diff
@@ -1,39 +1,47 @@
...
 - {{domxref("BatteryManager.levelchange_event", "levelchange")}}
-  - : 当电池电量（{{domxref("BatteryManager.level", "level")}} 属性）更新时被触发。
+  - : 当电源电量（{{domxref("BatteryManager.level", "level")}} 属性）更新时触发。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:44]:
```suggestion
  - : 当电池电量（{{domxref("BatteryManager.level", "level")}} 属性）更新时触发。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 10]
```diff
@@ -1,39 +1,47 @@
...
 
 - {{domxref("BatteryManager.charging")}} {{ReadOnlyInline}}
-  - : 一个布尔值，说明当前电池是否正在充电。
+  - : 一个指示当前电池是否正在充电的布尔值。
 - {{domxref("BatteryManager.chargingTime")}} {{ReadOnlyInline}}
-  - : 一个数字，代表距离充电完毕还需多少秒，如果为 0 则充电完毕。
+  - : 一个表示电池距离充电完毕还需多少秒的数字，或为 0 则表示电池已充电完毕。
 - {{domxref("BatteryManager.dischargingTime")}} {{ReadOnlyInline}}
-  - : 一个数字，代表距离电池耗电至空且系统挂起需要多少秒。
+  - : 一个表示距离电池耗电至空且系统挂起需要多少秒的数字。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/batterymanager/index.md:25]:
```suggestion
  - : 一个表示距离电池完全放电且系统挂起需要多少秒的数字。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 11]
```diff
@@ -1,39 +1,47 @@
...
 slug: Web/API/BatteryManager
+l10n:
+  sourceCommit: 8d54a21ae2677dba11569e7b7d918eac828af0b3
 ---
 
-{{APIRef}}
+{{ApiRef("Battery API")}}{{securecontext_header}}
 
-[电池状态 API](/zh-CN/docs/Web/API/Battery_Status_API) 中的 `BatteryManager` 接口提供了系统电池电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，该 promise 在完成状态时将返回一个 `BatteryManager` 接口。
+{{domxref("Battery Status API", "", "", "nocode")}} 的 **`BatteryManager`** 接口提供了系统电池电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，其兑现一个 `BatteryManager` 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/batterymanager/index.md:10]: 我们遵从 [W3C](https://www.w3.org/zh-hans/news/2014/last-call-battery-status-api/) 的翻译即可：

```suggestion
{{domxref("Battery Status API", "电池状态 API", "", "nocode")}} 的 **`BatteryManager`** 接口提供了系统电池电量的一些信息。{{domxref("navigator.getBattery()")}} 方法返回一个 promise 对象，其兑现一个 `BatteryManager` 接口。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 12]
```diff
@@ -1,39 +1,47 @@
...
+
+自 Chrome 103 起，{{domxref("Battery Status API", "", "", "nocode")}} 的 `BatteryManager` 接口只在严格上下文中暴露。
```
Comment (yin1999) [inline files/zh-cn/web/api/batterymanager/index.md:12]:
```suggestion
自 Chrome 103 起，{{domxref("Battery Status API", "电池状态 API", "", "nocode")}} 的 `BatteryManager` 接口只在安全上下文中暴露。
```

File: files/zh-cn/web/api/batterymanager/index.md [hunk 13]
```diff
@@ -45,5 +53,5 @@ _方法继承自其父接口_：{{domxref("EventTarget")}}。
 
 ## 参见
 
-- [电池状态 API](/zh-CN/docs/Web/API/Battery_Status_API)
-- {{domxref("navigator.getBattery")}}
+- {{domxref("Battery Status API", "", "", "nocode")}}
```
Comment (yin1999) [inline files/zh-cn/web/api/batterymanager/index.md:56]:
```suggestion
- {{domxref("Battery Status API", "电池状态 API", "", "nocode")}}
```

**Review by yin1999 — COMMENTED** [review]
关于 API 名称翻译的建议：

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**